### ISSUE TITLE
Cycle 0 list = [] ?

### ISSUE THREAD

**Issue by bitterjug** [elm-community/list-extra#149]
If the cycle length is zero, shouldn't we be returning a list of length zero?
https://github.com/elm-community/list-extra/blob/7ae12a5da9c1337188b415d70bc0d49df2db61b3/src/List/Extra.elm#L205

**Comment by Chadtech** [comment]
Yes, and I think that is the case, because `cycleLength` comes from the `List.length` of input list. So when `cycleLength` is zero, the input list is returned, which was necessarily an empty list.
